src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/JS/WasmTestCommand.cs

```diff
@@ -31,6 +32,34 @@ public WasmTestCommand() : base("test", true, CommandHelp)
         {
         }
 
+        static readonly string[] s_extensions = { ".cmd", ".exe" };
+
+        private static string GetFullPathTo(string engineBinary)
+        {
+            if (Path.IsPathRooted(engineBinary))
+                return engineBinary;
+
+            var path = Environment.GetEnvironmentVariable("PATH");
+
+            if (path == null)
+                return engineBinary;
+
+            foreach (var folder in path.Split(";"))
```

Comment:
I believe this separator differs per platform. There should be something like `Path.PathSeparator` or similar that would give you the right one

Reply:
This is only used on windows, it will be cleaner though.